cmd/podman/shared/create.go

```diff
@@ -615,6 +621,7 @@ func ParseCreateOpts(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 		ImageVolumeType:   c.String("image-volume"),
 		CapAdd:            c.StringSlice("cap-add"),
 		CapDrop:           c.StringSlice("cap-drop"),
+		Cgroup:            c.String("cgroup"),
```

Comment:
Can we name the option CgroupMode or similar, so there's no confusion? As written, it sounds like we're setting the actual CGroup the container will join

Reply:
I've followed the naming we used for other namespaces, e.g. have `--pid`, `--uts`, `--userns`.